services/user.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = "",
+        first_name: str = "",
+        last_name: str = "",
+) -> None:
+    get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
+        last_name=last_name
+    )
+
+
+def get_user(user_id: int) -> [User, str]:
+    try:
+        return get_user_model().objects.get(id=user_id)
+    except Exception as e:
+        return e
+
+
+def update_user(
+        user_id: int,
+        username: str = "",
+        password: str = "",
+        email: str = "",
+        first_name: str = "",
+        last_name: str = "",
+) -> [None, str]:
+    try:
+        user = get_user_model().objects.get(id=user_id)
+    except Exception as e:
+        return e
```

Comment:
the same here) 
and You can use here the get_user() function to simplify your code)